app/templates/index.hbs

```diff
@@ -2,7 +2,7 @@
     <div class="container layout">
       <div class="lg:col-5">
         <h1 class="text-hero-xl" id="hero-title">George Floyd</h1>
-        <p class="text-hero-base mb-2">Natosha McDade, Yassin Mohamed, Finan H. Berhe, Sean Reed, Steven Demarco Taylor,
+        <p class="text-hero-base mb-2">Tony McDade, Yassin Mohamed, Finan H. Berhe, Sean Reed, Steven Demarco Taylor,
           Breonna Taylor, Ariane McCree, Terrance Franklin, Miles Hall, Darius Tarver, William Green, Samuel David
           Mallard, Kwame Jones, De’von Bailey, Christopher Whitfield, Anthony Hill, De’Von Bailey, Eric Logan, Jamarion
```

Comment:
I noticed that De'Von was mentioned twice.
```suggestion
          Mallard, Kwame Jones, Christopher Whitfield, Anthony Hill, De’Von Bailey, Eric Logan, Jamarion
```

Reply:
@ijlee2 Thanks for catching that. Can you submit a separate PR since I already merged this one? thanks!

Reply:
Yep, will do.